### DISCUSSION TITLE
Option to have it enabled/disabled by default

### DISCUSSION THREAD

**Discussion by MarkyJD** [moalamri/vscode-inline-fold#52] [Ideas]
Thanks for your work. This extension makes working with Tailwind immensely better!

I am using the toggle command with a custom keybinding which is great, but I prefer to have it off by default, and then I use it sporadically as I need it. 

I would love an option to have it launch as disabled by default, to save me needing to toggle it off every time I launch vs code (I am lazy lol).  

Cheers

**Comment by moalamri** [comment]
Glad you found it useful 👍

I've pushed a branch that has the request feature. You can follow it up in here #53

I'm still hesitant about either I added other extra features I've been making for the next major release. Or to add your separately.

Thanks for the suggestion 👍

**Reply by moalamri** [reply]
Added in release v0.1.8
